.github/workflows/e2e.yaml

```diff
@@ -35,10 +35,10 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-gradle-
 
-    - name: Use Node.js 16
+    - name: Use Node.js 18
```

Comment:
If you're switching to Node 18 here then you should update it in the other workflows and the Dockerfile.

Reply:
Good catch! I definitely would not have remembered the Dockerfile.